datafiles/templates/Html/analytics-pixels-page.html.st

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+<head>
+$hackageCssTheme()$
+<title>Analytics pixels for $pkgname$ | Hackage</title>
+</head>
+
+<body>
+$hackagePageHeader()$
+
+<div id="content">
+
+<h2>Adding a analytics pixel to <a href="/package/$pkgname$">$pkgname$</a></h2>
+
+<p> 
+  Configure an analytics pixel to be automatically loaded on your package’s page on Hackage. 
+  You’ll need an image URL from any external analytics provider, e.g. <a href="about.scarf.sh">Scarf</a>, which is provided 
+  for free and can surface information about web traffic to your package including geographic 
+  distribution, version distribution, and companies.
```

Comment:
This text, which explicitly directs users to a specific service and provides promotional copy for it, is not going to work. We can merge without changes here, and I'll just modify it in a subsequent commit, if you prefer. But just wanted to be upfront about that. Ditto for the same text in the other template.

Reply:
I removed the references!